app/tests/test_search.py

```diff
@@ -739,7 +739,7 @@ def test_search_transferring_body_with_table_data_links(
                 ],
             ),
             (
-                "query=docx&sort=consignment-reference-desc",
+                "query=docx&sort=consignment_reference-desc",
```

Comment:
why did the sort field change from hyphen to underscore?

Reply:
it was wrongly applied , please refer the image

![image](https://github.com/nationalarchives/da-ayr-beta-webapp/assets/107114665/9fd238dc-d0ee-4a8c-8360-4163334e8e5b)

Reply:
oh I see, this suggests that this test isnt testing anything useful if the assertions were passing before and continue to pass with this change. We should probably update it to be more useful

Reply:
removed series and consignment reference sorting order test cases as they are not performing anything